api/src/pcapi/scripts/offer/add_gtl_id_to_offer.py

```diff
@@ -0,0 +1,304 @@
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '926' THEN '11020000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '927' THEN '11000000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '928' THEN '11000000'
+                                    ELSE '11000000'
+                                END
+                        WHEN offer."jsonData" ->> 'musicType' = '930'
+                            THEN
+                                CASE
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '931' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '932' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '933' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '934' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '935' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '936' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '937' THEN '02060000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '938' THEN '02060000'
+                                    ELSE '02060000'
+                                END
+                        WHEN offer."jsonData" ->> 'musicType' = '1000'
+                            THEN
+                                CASE
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '1001' THEN '17040000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '1002' THEN '09020000'
+                                    WHEN offer."jsonData" ->> 'musicSubType' = '1006' THEN '09020000'
+                                    ELSE '09000000'
+                                END
+                        ELSE '19000000'   
+                    END)              
+                ,
+                true
+            ) WHERE offer.id between :start and :end
+              AND offer."subcategoryId" in ('CONCERT', 'EVENEMENT_MUSIQUE',
+                'LIVESTREAM_MUSIQUE', 'ABO_CONCERT', 'FESTIVAL_MUSIQUE',
+                'SUPPORT_PHYSIQUE_MUSIQUE_CD', 'SUPPORT_PHYSIQUE_MUSIQUE_VINYLE',
+                'TELECHARGEMENT_MUSIQUE', 'ABO_PLATEFORME_MUSIQUE', 'CAPTATION_MUSIQUE')
+              AND offer."jsonData" is not null AND offer."jsonData" ->> 'musicType' is not null
+            """,
+            params={"start": i, "end": i + BATCH_SIZE},
+        )
+        if dry_run:
+            db.session.rollback()
+        else:
+            db.session.commit()
+
+        elapsed_per_batch.append(int(time.perf_counter() - start_time))
+        eta = _get_eta(end_id, start_id, elapsed_per_batch)
```

Comment:
```suggestion
        eta = _get_eta(end_id, i, elapsed_per_batch) 
```
Vu que tu es sur le i^ème batch ?